docs_dev/releasing.md

```diff
@@ -150,12 +151,9 @@ gcloud config set project kubeflow-releasing
 gcloud container clusters get-credentials kubeflow-releasing --zone us-central1-a --project kubeflow-releasing
 ```
 
-Use [this script](https://github.com/jlewi/kubeflow-dev/blob/master/create_context.sh) to set up your context properly:
-
-```
-create_context.sh $(kubectl config current-context) kubeflow-releasing
-```
+## Authorization to Publish a Release
```

Comment:
How about creating a section pre-requisites for creating a release

Reply:
done